source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -67,6 +67,8 @@ using the API.
 
 For example, you could automatically fetch data about the outcome of payment journeys. You could import that into your finance system so payments can be reconciled against bank transaction information.
 
+If you [generate a list of payments](/reporting/#generate-a-list-of-payments-search-payments) at the end of each day, you should use a `to_date` of at least one hour before you generate the list. This is because the API may not return up-to-date information about payment events from the last hour.
```

Comment:
This is correct, but I think, based on conversations with LAs, that they think about it the other way round. ie they know what time they want the to_date to be because it's set for other systems that rely on this data (ie midnight, or 5PM) and they need to run the report an hour later (ie at 1AM or 6PM). Rather than saying, you want to run a report at midnight so set the time for 11PM. Does that make sense...?

Reply:
Good point, thanks @miriamalice! I've clarified in the latest commits (and tweaked a bit for clarity).